### DISCUSSION TITLE
Reduce the cost of pvp items

### DISCUSSION THREAD

**Discussion by UndoUreche** [chromiecraft/chromiecraft#7276] [Suggestions]
I want to start with the mention that I'm talking about both honor and arena gear here, but for different reasons.

The structure of pvp in wow is similar to pve i.e. it's a ladder. Bgs give you starting gear, arena gives you the good stuff, and you need the bg gear to survive in arena.

### BG argument
Unfortunately tbc doesn't recycle the previous arena season's gear into honor gear so it makes catching up hard. Thankfully the staff added it to the arena vendor (cheers to you). There is however one issue: **this is meant to be a catch-up mechanic.** And that's the argument for honor gear. Honor gear costs over half of the honor cap per item. That's insanely pricey for gear that's supposed to get people ready for arena, and it turns bgs into a massive grind, prolonging the period where a new player gets mercilessly krumped, for no good reason.

I know custom vendor gear should be expensive, because it's either that or the custom vendor shouldn't exist. But this philosophy contradicts the nature of this catch-up mechanic, so please make an exception for this one.

### Arena argument
This one is plain and simple: there's not enough teams, there's not enough rating going around, there's not enough matches happening, and there won't be any new players trying it out if it takes some 28 weeks to finish getting your 5 pieces (not even counting the weapons) on the measly 300 points most low rated teams are going to get. This is a massive grind that nobody wants to get into.

---

Yes, I know the argument that pvp is its own way of playing the game and some people don't like it. But after urging some folks to give it a try (folks that have enjoyed it in the past) I've seen more than my fair share of them giving it up because the barrier to entry is too high.

**Comment by heyitsbench** [comment]
For the record, a similar solution was proposed here: https://github.com/chromiecraft/chromiecraft/discussions/7183 which was well received by the PvP community, but nothing has come from staff on it aside from a few comments in Discord saying they'd be willing if the prices were agreed upon.

**Reply by UndoUreche** [reply]
Then I did a bad job searching for similar suggestions. Thanks for linking it
